app/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="org.piwigo"
     android:installLocation="auto">
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+
+    <uses-permission android:name="android.permission.WRITE_SETTINGS"
```

Comment:
What do you use this permission for? We have to explicitly mention the permissions in the privacy policy according to googles requirements. So if we can avoid it it would be simpler, if not we have to check the privacy policy.

Reply:
This permission is for Brightness. Should i remove it ?

Reply:
Let me turn it around: I think we should keep it IFF we are confident, that we can do improve the UX if we have a setting in our APP, over just letting the user configure the brightness on system level.

So far _I_ do not see a benefit of having brightness adjustment in the settings. If you see it don't hesitate to explain, if not maybe it is better to remove it. Yes, it's a pity to waste the effort you invested, but we also have to keep in mind that every code that is there has to be maintained and increases the overall complexity, and that's why I'd like to have only code that is there for a good reason.

I hope you understand, it's nothing against you, I really appreciate your work and your outstanding responsiveness!

Reply:
@ramack Ok, no problem. Still new to GitHub :D. Thanks for your advice. Will remove the brightness.